documentation/guides/introduction/01_quick_start.md

```diff
@@ -0,0 +1,169 @@
+
+You can now use your deployed token program!  
+
+:::tip
+If you did not have enough funds to deploy a program, you can use the `token_quickstart.aleo` program to test the execution of the token program methods.
+:::
+
+When you call a method from your program, the logic is executed locally and is accompanied with a zero-knowledge proof that attests to the correctness of the program execution and its corresponding outputs.  This proof does not reveal any information about the input values.  If the proof is verified by the validators, then the blockchain state is updated.  
+
+* Minting a private token
+
+Minting a private token entails creating a Record.  Records are data structures that are stored on-chain as ciphertexts and can only be decrypted using the View Key, which is derived from the Record owner's Private Key.  This ensures that only the owner of a Record has the ability to decrypt it.  
+
+To mint a private token, navigate to the Execute widget in the Leo Playground and enter the name of your program in the Program ID tab.  Alternatively, you can use the deployed token program 'token_quickstart.aleo`.  Click the magnifying glass icon to confirm that the Program ID corresponds to a deployed program.
+
+![execute](images/deploy.png)
+
+Next, enter your Private Key and select the `mint_private` method from the Function drop-down menu.  In the Inputs tab, enter your Account Address and a `u64` integer.
+
+```
+["ALEO_ADDRESS_HERE", "50u64"]
+```
+Click Estimate Fee followed by Execute.  As before with the deploy transaction, you can copy and past your transaction ID in a block explorer.  You should see something resembling the following:
+
+![mint_private](images/mint_private.png)
+
+:::info
+The output of the transaction is a Record.  In order to decrypt the Record, you need to copy and past the ciphertext, navigate to the Records widget in the Leo Playground, enter the Record ciphertext along with the View Key that corresponds to your Account's Private Key, and click Decrypt.  The decrypted Record should resemble the following:
+```{
+  owner: aleo1kypwp5m7qtk9mwazgcpg0tq8aal23mnrvwfvug65qgcg9xvsrqgspyjm6n.private,
+  amount: 50u64.private,
+  _nonce: 3175255370513411091535466147458245312227668453916963245036391157478647265587group.public
+}
+```
+To learn more about Records, click [here](../../concepts/fundamentals/02_records.md)
+:::
+
+* Mint a public token
+
+Public tokens are stored in mappings.  Unlike Records, which are encrypted on-chain, mappings will always contain public data.  
+
+Navigate back to the Execute widget and repeat the steps in the previous section for minting a public token.  The transaction summary from the block explorer should resemble the following:
+
+![mint_public](images/mint_public.png)
+
+Notice that this transaction produces a Future instead of a Record.  A Future encapsulates a set of instructions for validators to execute on-chain, such as updating the values in a mapping.  
```

Comment:
```suggestion
Unlike the `mint_private` function, the `mint_public` function creates an execution transaction that contains a `Future` instead of a `Record`.  A `Future`  contains a set of instructions for validators to execute on-chain which can read or update mappings.  If the `mint_public` execution transaction is valid, it will be accepted by the validators which will run the instructions in the `mint_public` `Future` that update the `account` mapping.

This account mapping can then be queried publicly by anyone.
```